src/openforms/forms/tests/test_import_export.py

```diff
@@ -399,3 +400,223 @@ def test_import_form_with_category(self):
+            dedent(
+                """
+                Some confirmation email content translation with the obligatory
+                {% appointment_information %}
+                {% payment_information %}
+                """
+            ).strip(),  # trailing newline
+        )
+        self.assertEqual(
+            imported_form.confirmation_email_template.content,
+            dedent(
+                """
+                Untranslated confirmation email content with the obligatory
+                {% appointment_information %}
+                {% payment_information %}
+                """
+            ).strip(),  # trailing newline
+        )
+
+        self.assertEqual(
+            imported_form_step.form_definition.name, "Untranslated form definition name"
+        )
+        self.assertEqual(
+            imported_form_step.form_definition.name_en,
+            "Some form definition name translation",
+        )
+        self.assertEqual(form_step.previous_text, "Untranslated previous step text")
+        self.assertEqual(form_step.save_text, "Untranslated save step text")
+        self.assertEqual(form_step.next_text, "Untranslated next step text")
+
+        # inspect the exported data structure, configs etc.
+        imported_json = form_to_json(imported_form.pk)
+        expected_differences = {
+            "forms": [
+                "uuid",  # import gets assigned a new one
+                "url",  # contains uuid
+                "active",  # import gets set to inactive
+                "category",  # gets unset
+                "confirmation_email_template",  # TODO bug???
+                "steps",  # duplicated in formSteps
+            ],
+            "formSteps": [
+                "form_definition",  # is a url with uuid
+                "uuid",
+                "url",
+            ],
+            "formDefinitions": [
+                "uuid",
+                "url",
+            ],
+        }
+
+        def remove_expected_differences(part: str, obj1: dict, obj2: dict) -> None:
+            for field in expected_differences.get(part, ()):
+                del obj1[field]
+                del obj2[field]
+
+        def head(v: list | dict) -> dict | None:
+            "Return first dict in the list"
+            if isinstance(v, dict):  # _meta is already an object
+                return v
+            return None if not v else v[0]
+
+        # assert JSON is still equivalent
+        for part, json_string in original_json.items():
+            expected_data = json.loads(json_string)
+            imported_data = json.loads(imported_json[part])
+            # they should have the same number of elements/keys to start with
+            self.assertEqual(len(imported_data), len(expected_data))
+
+            expected = head(expected_data)
+            imported_value = head(imported_data)
+
+            remove_expected_differences(part, expected, imported_value)
+            self.assertEqual(imported_value, expected)
```

Comment:
yeah when/if this is failing you'll really want a `with self.subTest(part=part)` around it to better grok the structure